onnx2torch2/node_converters/pad.py

```diff
@@ -123,7 +123,14 @@ def _(node: OnnxNode, graph: OnnxGraph) -> OperationConverterResult:  # pylint:
     pads_name = node.input_values[1]
     pads = []
     if pads_name in graph.initializers or pads_name in graph._node_output_values:  # pylint: disable=W0212
-        pads = get_const_value(pads_name, graph).tolist()
+        # try and get padding arg value
+        try:
+            pads = get_const_value(pads_name, graph).tolist()
+        except KeyError as e:
+            raise ValueError(
+                f"Dynamic padding is not supported. Pad node `{node.name}`"
+                f"has padding arg `{pads_name}` with no static value. Error: {e}"
+            ) from e
```

Comment:
We shouldn't reveal anything about the internal errors, you have already displayed the node's name and the arg creating the issue.
```suggestion
        except KeyError:
            raise ValueError(
                f"Dynamic padding is not supported. Pad node `{node.name}`"
                f"has padding arg `{pads_name}` with no static value.")
```